Source/Device.swift.gyb

```diff
@@ -331,6 +672,8 @@ public enum Device {
       case .simulator(let model): return model.diagonal
       case .unknown: return -1
       }
+    #elseif os(macOS)
+    return -1
```

Comment:
```suggestion
      return -1
```

Reply:
I will do it to be compliant with the switch, but I think there is some issues with formatting
because now #if #elseif do not shift the code anymore

Reply:
formatting done ✅